content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md

```diff
@@ -0,0 +1,135 @@
+
+___Huaweis previous flagship record is held by the P9 which sold 12 million units within it's first 11 months. The only datapoint we got for P10 so far, shows it's selling 36% better than the P9 did at that point in time. As Huawei have seen a overall 20.6% increase in 2017H1 on total unit shipments (all models), and P10 seemingly a very big success with the consumers, I'd say a _modest_ 25% increase over P9 could be realistic. That will put P10 sales after 11 months on market somewhwere around 15-16 million units. Competition will be harder this autumn than the last, because of the anniversary iPhone, and a Samsung Note 8 that so far haven't exploded.___
+
+### Honor 9
+This is tough. Detailed sales about any Honor model are hard to come by. We know that yearly sales of the Honor subsidiary accounted for approximately 60 million units of Huawei's total 139.3 shipped units.  Other than that, the only data available is that Honor 8 sold 1.5 million units by it's second month after release, while the Honor 9 (with Vidhance) have exceeded 1 million during it's first month on market. These numbers seem to be in line with the _Huawei Mate 8_ and the _Huawei P8 Lite_. Which respectively sold 7 million units at 12 months for the Mate, and 10 million units at 9 months for the P8. That's quite a span. 
+
+__Based on strong year-on-year growth for the Honor brand, and the "Honor 9" realtively cheap chinese price, compared to the P10, one could expect this to sell a lot of units. But it got a lot of competition from other Huawei / Honor models. Earlier this year, Honor released the "Honor V9" in china, which was rebranded into the "Honor 8 Pro" in international markets. It's specifications and price is pretty equal to both the Honor 9, and the Huawei P10. So quite a lot of potential Honor 9 customers, probably end up with a "Honor V9 / Honor 8 Pro" or a P10 instead. As we know P10 sales look incredible strong so far, my guess is that the Honor 9 will sell somewhere around 8-10 million units in it's first 12 months. But I'm feeling less confident in this prediction than for the Mate 9 and P10.__
+
+## Conclusion
+The current models from Huawei / Honor with Vidhance included is
+* Mate 9 family
+* P10 family
+* Honor 9
+
+> __I expect these three models to combined sell between 33 to 38 million units during their first 12 months__.
+
+That is counting each model from it's release, and 12 months out. They are likely to continue selling some millions (combined) even after 12 months, but a lack of good datapoints at this make it hard to predict from after 12 months.
+
+---
```

Comment:
This line makes no sense as the links below actually will show up at their usage.